README.md

```diff
@@ -970,6 +970,7 @@ Parameters:
  - selector (required) `String|String[]` – DOM-node selector that you need to capture
  - opts (optional) `Object`:
    - ignoreElements (optional) `String|String[]` – elements, matching specified selectors will be ignored when comparing images
+   - tolerance (optional) `Number` – maximum allowed [CIEDE2000](http://en.wikipedia.org/wiki/Color_difference#CIEDE2000) difference between colors. See [browsers.tolerance](#tolerance)
```

Comment:
Please, don't copy-paste tolerance description here. Just tell that this option overwrites tolerance from config for given assertView and give a link to base tolerance description